### PR TITLE
Jasmine Allen Elizabeth remove first substring

### PR DESCRIPTION
**Pull request by qlychee**

### CODE REVIEW

**Comment by derek-read** [comment]
No changes need to be made. The function does what it is supposed to do and removes the first substring matching the given substring to find.

**Comment by nzimmermann** [comment]
Function works as described. Consider perhaps letting it also remove one space " " so that it looks nicer at the end.

**Review by derek-read — COMMENTED** [review]
No changes need to be made. The function does what it is supposed to do and removes the first substring matching the given substring to find.

**Comment by joedertie** [comment]
This looks good, but may be a good idea to add small comment describing function in the header file and author name above the function in ManyFunctions.cpp file.

File: utility_functions/ManyFunctions.h [hunk 1]
```diff
@@ -18,5 +18,7 @@
 */
 //int Sum(int num1, int num2);
```
Comment (joedertie) [inline utility_functions/ManyFunctions.h:21]: May be a good idea to add a comment here.

File: utility_functions/main.cpp [hunk 2]
```diff
@@ -1,5 +1,15 @@
 #include "ManyFunctions.h"
+#include <string>
+#include <vector>
+#include <iostream>
+
 
 int main() {
     // make sure to run your functions!
+    std::string old_string = "How are you?";
+    std::string find_string = "are";
```
Comment (nzimmermann) [inline utility_functions/main.cpp:22]: Adding a space to the find_string could make the end code look nicer

**Review by joedertie — DISMISSED** [review]

**Review by nzimmermann — COMMENTED** [review]
Looks good

**Review by muzny — APPROVED** [review]

**Review by muzny — APPROVED** [review]

**Pull request merged**